data/calculated_parser/functions.py

```diff
@@ -226,6 +227,131 @@ def linearInterp(x1, y1, x2, y2, x):
 
     return cd_depth
 
+
+def __is_min(p1, p2, p3):
```

Comment:
Is this function being used?

Reply:
No, switched to using scipy extrema function